Tasks/AzureMysqlDeployment/models/FirewallConfiguration.ts

```diff
@@ -0,0 +1,17 @@
+export class FirewallConfiguration {
+
+    private ipAdressAlreadyAdded: boolean;
```

Comment:
IPAddressAlreadyAdded property is being computed from other property. So we don't need this.

Reply:
We are keeping IPAddressAlreadyAdded and IP address to be separate.